core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -526,7 +528,17 @@ private Certificate fetchEphemeralCertificate(KeyPair keyPair) {
     if (enableIamAuth) {
       try {
         credentials.get().refresh();
```

Comment:
Can we refresh after we downscope?

Reply:
Unfortunately no,  we get a nullpointer if we do that.